### PR TITLE
Adds Brazilian portuguese translation file.

### PR DESCRIPTION
**Pull request by victorpierredev**
Adds Brazilian Portuguese translation file. 65% complete. Missing translations will be added later.

### CODE REVIEW

**Comment by akazorg** [comment]
Muito obrigado!
I'll reuse it to start the PT version.

Please @vpakg, take special attention to the capitalized words, they are important for the UI.

**Comment by victorpierredev** [comment]
@akazorg Thanks! Duly noted. I've translated another chunk this morning, will probably be able to finalize it tonight. I'll try to keep the capitalizations consistent with the English version. Although, and please feel free to disagree, in both English and French (and possibly all the romantic languages), capitalization and punctuation work a bit different than English. 

I work building bilingual sites on a regular basis (French and English) and even though we have capitalized letters in English (e.g. Lorem Ipsum Dolor Sit Amet) when translating to French we always leave just the first word capitalized (e.g. Lorem ipsum dolor sit amet). 

But maybe this is a particularity of the company I work for or the French language.

**Comment by akazorg** [comment]
Hi @vpakg, absolutely about the differences between languages.

I meant for the capitalized headers and buttons. If we apply a rule and be consistent, the UI / UX will work perfectly.

I've also translated a part of PT (your PR help, ofc!), must be ready soon.

**Comment by akazorg** [comment]
@vpakg I've just finished PT, if you want to reuse it, [check here](https://github.com/the-control-group/voyager/blob/i18n/publishable/lang/pt/voyager.php).

File: publishable/lang/pt_br/voyager.php [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    'generic' => [
+        'action'                 => 'Ação',
+        'actions'                => 'Ações',
+        'add'                    => 'Adicionar',
+        'add_folder'             => 'Adicionar pasta',
+        'add_new'                => 'Adicionar novo',
+        'all_done'               => 'Concluido',
+        'are_you_sure'           => 'você tem certeza',
+        'are_you_sure_delete'    => 'Tem certeza de que deseja excluir
+',
```
Comment (marktopper) [inline publishable/lang/pt_br/voyager.php]: Please remove newline

**Review by marktopper — DISMISSED** [review]
It also seems that there in the english translation file exists a `database`, `dimmer`, `form`, `datatable`, `theme`, `json`, `analytics` and a `error` group. 😄 
If you have time it would be nice to get those added as well. However, great job.

**Comment by victorpierredev** [comment]
Hey, @marktopper and @akazorg thanks for the input! Realy sorry this is taking so long, I'm crazy busy these days. I'll get this done today for sure. If anything, ping me on Slack, I'm usually online there. 

Cheers.

**Comment by victorpierredev** [comment]
Reworked it, pushed `patch-2` for review.

**Comment by akazorg** [comment]
Looks great Vitor, will merge it.
Thank you!

**Pull request merged**